crypto/signer_test.go

```diff
@@ -67,14 +67,17 @@ func TestSigner(t *testing.T) {
 		{message: []byte("message")},
 	} {
 		sig, err := signer.Sign(test.message)
+		if err != nil {
+			t.Errorf("Failed to sign log root: %v", err)
```

Comment:
nit: Add a continue here? If signing failed there isn't much point in progressing the test.